code/modules/reagents/Chemistry-Holder.dm

```diff
@@ -198,9 +199,10 @@ datum/reagents/proc/metabolize(var/mob/M)
 	for(var/A in reagent_list)
 		var/datum/reagent/R = A
 		if(M && R)
-			if(M.reagent_check(R) != 1)
+			if(M.reagent_check(R) != 1 && last_tick == 3)
 				R.on_mob_life(M)
-
+				last_tick = 0
+			last_tick++

```

Comment:
This has the wrong indent level. You'll be counting once per reagent, and only if it actually processes, which is definitely not what you want. You should move it two indent levels down.